distributed/tests/test_cancelled_state.py

```diff
@@ -536,6 +536,16 @@ async def release_all_futures():
                 (f3.key, "executing", "released", "cancelled", {}),
                 (f3.key, "cancelled", "fetch", "resumed", {}),
                 (f3.key, "resumed", "memory", "memory", {}),
+                (
```

Comment:
The resumed task should be rejected by the scheduler because its run ID is stale, which triggers the task to be released and recomputed.